### ISSUE TITLE
[Feature Request] Make errors less mysterious.

### ISSUE THREAD

**Issue by phonexicum** [ffuf/ffuf#340]
Sometimes the fuzzing process will collect a couple of errors. And the reason is always the matter of interest.
It would be very helpful to have a flag which will force FFUF to report minimal information about an error in a single string, e.g.:
The input (from wordlist), the error reason (e.g. networking connection timeout or smth other), maybe the error message description.

Thanks for the FFUF tool, it is awesome.

**Comment by bsysop** [comment]
Hi @phonexicum, indeed it is very important, but exists with -debug-log option, you can save a file with all that critical errors!

Happy hacking!

**Comment by phonexicum** [comment]
Indeed. Well, I have looked for something less verbose.
In a day-to-day usage it is not so comfortable to save a separate file with output for every FFUF run and grep it for errors. It will be more convenient to have ability to include errors into "relevant" output, which operator will look through afterwards.

Anyway now it become the matter of taste, and therefore feel free to close the issue if you find it useless.

**Comment by bsysop** [comment]
Yeah, for a small and controlled fuzzing, would be nice have it, for fuzzing millions of targets, would be crazy. So it really depends the approach.

joohoi will see it later and decide!

Happy hacking.

**Comment by joohoi** [comment]
Yeah. It's a double edged sword. Printing out the errors will usually result into the terminal to be completely spammed with errors because of the speed. The decision to hide the errors is a deliberate. Maybe this could be turned into a feature request to add `stdin` option for `--debug-log` flag?

**Comment by negatratoron** [comment]
You can pass `/dev/stdout` or `/dev/stderr` as the argument to `-debug-log` to have the debug log output there.
